sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/SearchIndexerAsyncClient.java

```diff
@@ -233,19 +236,20 @@ public String getEndpoint() {
      * help with debugging.
      * @return a list of DataSources
      */
-    public PagedFlux<SearchIndexerDataSource> listDataSources(RequestOptions requestOptions) {
+    public PagedFlux<SearchIndexerDataSourceConnection> listDataSourceConnection(RequestOptions requestOptions) {
```

Comment:
This should be plural, i.e. `listDataSourceConnections`.

Reply:
Will check all naming and address in https://github.com/Azure/azure-sdk-for-java/pull/11590